tests/test_models.py

```diff
@@ -1613,17 +1613,17 @@ def test_validate_categories(self):
 
     def test_validate_bad_doi(self):
         with assert_raises(ValidationError):
-            Node(preprint_doi='nope').save()
+            Node(preprint_article_doi='nope').save()
         with assert_raises(ValidationError):
-            Node(preprint_doi='https://dx.doi.org/10.123.456').save()  # should save the bare DOI, not a URL
+            Node(preprint_article_doi='https://dx.doi.org/10.123.456').save()  # should save the bare DOI, not a URL
         with assert_raises(ValidationError):
-            Node(preprint_doi='doi:10.10.1038/nwooo1170').save()  # should save without doi: prefix
+            Node(preprint_article_doi='doi:10.10.1038/nwooo1170').save()  # should save without doi: prefix
 
     def test_validate_good_doi(self):
         doi = '10.10.1038/nwooo1170'
-        self.node.preprint_doi = doi
+        self.node.preprint_article_doi = doi
         self.node.save()
-        assert_equal(self.node.preprint_doi, doi)
+        assert_equal(self.node.preprint_article_doi, doi)
 
```

Comment:
I know we are all "test the view not the model", but since you have a fancy new preprint model, it might be good to add some tests for that. Especially since you have permissions and stuff on it. This could be put on a new ticket if we're in a rush, but make it a high-priority Maintenance task and bump it to like spot 3-5 in the Maintenance backlog.